core_impl/src/main/java/com/google/instrumentation/stats/MeasurementDescriptorToViewMap.java

```diff
@@ -20,25 +20,27 @@
 import java.util.Map;
 
 /**
- * A class that stores a singleton map from {@link MeasurementDescriptor.Name} to {@link View}.
+ * A class that stores a singleton map from {@link MeasurementDescriptor.Name}s to
+ * {@link MutableView}s.
  */
 final class MeasurementDescriptorToViewMap {
 
   /*
    * A synchronized singleton map that stores the one-to-many mapping from MeasurementDescriptors
-   * to Views.
+   * to MutableViews.
    */
-  private final Multimap<MeasurementDescriptor.Name, View> mutableMap =
-      Multimaps.synchronizedMultimap(HashMultimap.<MeasurementDescriptor.Name, View>create());
+  private final Multimap<MeasurementDescriptor.Name, MutableView> mutableMap =
+      Multimaps.synchronizedMultimap(
+          HashMultimap.<MeasurementDescriptor.Name, MutableView>create());
 
   /**
-   * Returns a {@link View} corresponding to the given {@link ViewDescriptor}.
+   * Returns a {@link MutableView} corresponding to the given {@link ViewDescriptor}.
    */
-  final View getView(ViewDescriptor viewDescriptor) {
-    Collection<View> views = mutableMap.get(
+  final MutableView getMutableView(ViewDescriptor viewDescriptor) {
```

Comment:
Added a `getView()` method. However we need to keep `getMutableView()` as well, since `record()` needs to call it and retrieve the corresponding `MutableView`.